### PR TITLE
Fix documentation for current build process

### PR DESCRIPTION
**Pull request by jcreedcmu**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,85 +32,29 @@ Here are a few things you can do that will increase the likelihood of your pull
...
-##### From VS Code
-
-`Terminal > Run Build Task...` (or just `Ctrl+Shift+B` with the default key bindings)
-
-##### From the command line
+#### Building
 
 ```shell
-rush build --verbose
+gulp
```
Comment (aeisenberg) [inline CONTRIBUTING.md:52]:
```suggestion
npm run build
```

This prevents the need to have `gulp-cli` globally installed.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,85 +32,29 @@ Here are a few things you can do that will increase the likelihood of your pull
...
-
-A few more things to know about using rush:
-
-* Avoid running `npm` for any commands that install/link dependencies
-* Instead use the *rush* equivalent: `rush add <package>`, `rush update`, etc.
-* If you plan on only building via VS Code tasks, you don't need Rush installed at all, since those
-tasks run `common/scripts/install-run-rush.js` to bootstrap a locally installed and cached copy of
-Rush.
-
 ### Building
```
Comment (shati-patel) [inline CONTRIBUTING.md:35]: Nice to see the build process simplified 😃 

I find the current headings a bit confusing though. I can't add suggestions for some reason, but how about replacing lines 35-59 ("### Building" -> "### Installing") with something like this? 👇🏼 (If I've interpreted the instructions properly!) 

--- 
### Installing all packages

From the command line, go to the directory `extensions/ql-vscode` and run

```shell
npm install
```

### Building the extension

From the command line, go to the directory `extensions/ql-vscode` and run

```shell
npm run build
```

Alternatively, you can build the extension within VS Code via `Terminal > Run Build Task...` (or `Ctrl+Shift+B` with the default key bindings).

### Installing the extension

...

Comment (jcreedcmu) [reply CONTRIBUTING.md:35]: I like that better, thanks!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -32,85 +32,23 @@ Here are a few things you can do that will increase the likelihood of your pull
...
 
 ```shell
-rush rebuild --verbose
+npm run build
 ```
 
-Note that `rush rebuild` performs a complete rebuild, whereas `rush build` performs an incremental build and in many cases will not need to do anything at all.
+Alternatively, you can build the extension within VS Code via `Terminal > Run Build Task...` (or `Ctrl+Shift+B` with the default key bindings).
 
 ### Installing
```
Comment (shati-patel) [inline CONTRIBUTING.md:53]:
```suggestion
### Installing the extension
```

Perhaps clarify what we're installing (i.e. the extension itself, not installing packages etc)

**Pull request merged**